src/Tribe/Attendee_Registration/Template.php

```diff
@@ -20,34 +20,37 @@ class Tribe__Tickets__Attendee_Registration__Template extends Tribe__Templates {
 	public function hook() {
 
 		// Spoof the context
-		add_filter( 'the_posts', array( $this, 'setup_context' ), -10 );
+		add_filter( 'the_posts', [ $this, 'setup_context' ], -10 );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: e5f8028389c369b12cab1af91a098a05</sub>